Power BI time granularity docs (#8725)
fishtown-analytics/docs.getdbt.com: website/snippets/_sl-custom-granularity.md

<p>When you select time dimensions in the <strong>Group By</strong> menu, you'll see a list of available time granularities. The lowest granularity is selected by default. Metric time is the default time dimension for grouping your metrics.</p>

:::info 

> Note: [Custom time granularities](/docs/build/metricflow-time-spine#add-custom-granularities) (like fiscal year) aren't currently supported or accessible in this integration. Only [standard granularities](/docs/build/dimensions?dimension=time_gran#time) (like day, week, month, and so on) are available. If you'd like to access custom granularities, consider using the [Semantic Layer APIs](/docs/dbt-cloud-apis/sl-api-overview).

Reviewer: ```suggestion
Note: In the Power BI integration, [Custom time granularities](/docs/build/metricflow-time-spine#add-custom-granularities) (like fiscal year) aren’t currently available. Only [standard granularities](/docs/build/dimensions?dimension=time_gran#time) (like day, week, month, and so on) are available. If you'd like to access custom granularities, consider using the [Semantic Layer APIs](/docs/dbt-cloud-apis/sl-api-overview).
```

Reviewer: Hiya @mirnawong1 

This is a non-blocking suggestion, reading through the thread in slack (of this being a limitation of the Power BI connector), I thought it might be a good idea to explicitly mention Power BI.

Author: resolving this without committing as this applies to all integrations
